prac_05/hex_colours.py

```diff
@@ -0,0 +1,17 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Hexadecimal colour lookup
+"""
+
+COLOUR_CODES = {"DarkSalmon": "#E9967A", "DarkKhaki": "#BDB76B",
+                "Fuchsia": "#FF00FF", "SlateBlue": "#6A5ACD",
+                "MediumSpringGreen": "#00FA9A", "MediumAquamarine": "#66CDAA",
+                "MediumSlateBlue": "#7B68EE", "SandyBrown": "#F4A460",
+                "SaddleBrown": "#8B4513", "AntiqueWhite": "#FAEBD7",
+                "Tomato": "#FF6347", "DarkOrange": "#FF8C00", "HotPink": "#FF69B4",
+                "Crimson": "#DC143C", "IndianRed": "#CD5C5C"}
+
+colour = input("Enter a colour name: ")
+while colour != "":
+    print("The code for \"{}\" is {}".format(colour, COLOUR_CODES.get(colour)))
```

Comment:
No need for the escape characters in the print function in your case. If you need to use " " within the print string, you can't use " " around the whole thing. "    ""     " will cause an error. Try '   "{}"   '
print('The code for "{}" is {}'.format(colour, COLOUR_CODES.get(colour)))
print("The dog 'barked' at the cat") will have no errors
print('The dog "barked" at the cat') will have no errors
print("The dog "barked" at the cat") will Error... etc